src/partials/projects/ProjectStatus.vue

```diff
@@ -125,6 +126,11 @@
                                                 </span>
                                             </div>
                                         </div>
+                                        <AiAlert
+                                            v-if="!project.importStatus && project.projectCreationCompleted"
+                                            variant="warning"
+                                            text="The import status could not be retrieved. If the issue persists, please contact an administrator."
```

Comment:
I might be best to leave out the contact an administrator bit actually, just in case administrators get inundated with too many issues 👀
```suggestion
                                            text="Something went wrong when retrieving the study import status"
```